app/models/organization.rb

```diff
@@ -16,6 +16,43 @@ class Organization < ApplicationRecord
   validates :in_business, inclusion: { in: [CLOSED, OPEN, UNKNOWN] }
 
   include Auditable
+  include Searchable
+
+  searchable do
+    field :id, type: 'integer'
+    field :name, using: :names, type: 'string', analysis: FULLTEXT_ANALYSIS, factor: 1.0
```

Comment:
The `factor` is a relative field weighting applied only when using the default search lookup via the `#estella_search` interface.